src/bridge.ts

```diff
@@ -105,25 +105,30 @@ window.Notification = function (title: string, options: NotificationOptions) {
         body: options.body || "",
       };
 
+  const isSound = settings.get(SETTING_NOTIFICATION_SOUND,true) as boolean;
   notificationOpts.silent = true;
 
   const notification = new ElectronNotification(notificationOpts);
   notification.addListener("click", () => {
     app.mainWindow?.show();
     document.dispatchEvent(new Event("focus"));
+    if (!isSound)
+    {
+      (document.querySelector('a[href$="/' + options.data.id + '"]') as HTMLElement).click();
```

Comment:
an explanation of what this line is doing would be nice in a short comment or two